.github/dependabot.yml

```
@@ -17,3 +17,13 @@ updates:
     ignore:
       - dependency-name: '*'
         update-types: ['version-update:semver-patch', 'version-update:semver-minor']
+
+  - package-ecosystem: "gomod"
+    directory: "go"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "golang.org/x/mod"
+      - dependency-name: "golang.org/x/tools"
+    reviewers:
+      - "github/codeql-go-team"
```

```suggestion
      - "github/codeql-go"
```
I tried @-mentioning and this is what auto-completed.